packages/nextjs/src/tools/codegen/utils.ts

```diff
@@ -240,7 +242,13 @@ export const sendCode = async ({
     console.error(
       chalk.red(`Failed to send extracted code from ${file.path}: ${response.statusText}`)
     );
+    debug.http('Error details: %o', {
+      status: response.status,
+      text: await response.text(),
+      url: response.url,
+      headers: response.headers,
```

Comment:
```suggestion
      headers: Object.fromEntries(response.headers.entries()),
```

Reply:
from my experience `response.headers` is not a plain object and it's a Headers instance. It will log a unhelpful output, often just [Headers]. But if you think it does the job then you can ignore it.